thirdeye-ui/src/app/pages/impact-dashboard-page/data/use-impact-dashboard-api-requests.tsx

```diff
@@ -188,12 +207,22 @@ export const useImpactDashBoardApiRequests = ({
         }
     }, [recentAnomalyInvestigatedId]);
 
+    useEffect(() => {
+        if (alerts) {
+            const activeAlertIds = alerts
+                .filter((alert) => alert.active)
+                .map((alert) => alert.id);
+            getEnumerationItems({ alertIds: activeAlertIds });
```

Comment:
by the name of the function i am thinking if this is needed to be inside `useEffect` or otherwise ? what does this function do which to notify react to re-render ?

Reply:
its returned from a custom hook which takes care of the re-render on fetch